readers/docker/containers_cpu_linux.go

```diff
@@ -55,7 +55,7 @@ func (m *DockerContainersCpu) Run() error {
 
 			data, err := gopsutil_docker.CgroupCPU(containerDir, m.CgroupBasePath)
 			if err == nil {
-				m.Data[container.Image+"-"+container.ID] = data
```

Comment:
One more thing, the reason why the key is named this way is to ensure uniqueness. If we are changing to only use image name, then we need to include tag as well.